galleries/nxdata/plot_fscan2d.py

```diff
@@ -0,0 +1,89 @@
+"""
+==================
+2D Continuous scan
+==================
+
+Data on a regular two-dimensional grid which is the combination
+of an image and a histogram.
+
+The x-axis defines the bin edges along the first dimension.
+The y-axis defines the grid coordinates along the second dimension.
+
+This is a typical 2D scanning example where a sample is scanned through
+a focussed X-ray beam in two dimensions. The x motor is scanned continuously
+and both motors have an encoder which records the actual motor position as
+opposed to the requested or `set` position.
+
+.. code::
+
+  @NX_class = "NXroot"
+  @default = "scan1"
+  scan1:
+    @NX_class = "NXentry"
+    @default = "data"
+    data:
+      @NX_class = "NXdata"
+      @axes = ["x_set", "y_set"]
+      @x_encoder_indices = [0, 1]
```

Comment:
not registered among possible axes

Reply:
You mean `x_encoder`? 

That's exactly what I thought as well. Which lead to my changes in https://github.com/nexusformat/definitions/pull/1213 to be wrong as mentioned in https://github.com/nexusformat/definitions/issues/1381, the whole reason for the PR to begin with.

`@axes` gives the default axis to be used for each signal dimension. Hence this part in the new description

> Highlight consequences of these constraints:
> ...
> A list of all available axes is not provided directly. All strings in the axes attribute
> (excluding the “.” string) are axis field names. In addition the prefix of an attribute
> ending with the string “_indices” is also an axis field name.

Reply:
Resolved in Telco.  @sanbrock pointed out the section "Highlight consequences of these constraints" which address this comment.